### PR TITLE
Update `iCubGenova09` to the use of the latest NWS/NWC layers

### PR DESCRIPTION
**Pull request by pattacini**
cc @randaz81 @S-Dafarra @davidetome

### CODE REVIEW

**Comment by pattacini** [comment]
`iCubGenova09` has been successfully tested ✔️ 
Thanks to @randaz81 and @S-Dafarra!

I'll keep this open to receive:
- from @davidetome the mods to include also the files to start up single limbs
- from @S-Dafarra the mods to include the latest changes to mirror AMI's fork

**Comment by davidetome** [comment]
I've added the missing files as requested, please @randaz81 take a look to them 👍🏻 

cc @pattacini @S-Dafarra

**Comment by pattacini** [comment]
Thanks @davidetome 👍🏻

**Comment by S-Dafarra** [comment]
Thanks @davidetome for adding the missing files. While reading the changes, I realized that having multiple calibrator files for the arms might be risky. Right now, if we need to modify some calibration parameters of some fingers, we tend to modify the file ``calibrators/left_arm-calib.xml`` for example. With the other calibration files, we should duplicate the same modification to another file. Hence, the risk of forgetting or human error increases a lot.

Clearly, we had the same problem before, but at this point, I would exploit this occasion to avoid this kind of issue. Sorry for noticing it only after asking to resuscitate those files.

In short, I would kindly ask to delete the duplicate calibration files and the ``xml`` files that were needing them.

**Comment by davidetome** [comment]
> Thanks @davidetome for adding the missing files. While reading the changes, I realized that having multiple calibrator files for the arms might be risky. Right now, if we need to modify some calibration parameters of some fingers, we tend to modify the file `calibrators/left_arm-calib.xml` for example. With the other calibration files, we should duplicate the same modification to another file. Hence, the risk of forgetting or human error increases a lot.
> 
> Clearly, we had the same problem before, but at this point, I would exploit this occasion to avoid this kind of issue. Sorry for noticing it only after asking to resuscitate those files.
> 
> In short, I would kindly ask to delete the duplicate calibration files and the `xml` files that were needing them.

Ciao @S-Dafarra , I think you refer to forearms calibrators.
Honestly, I simply ported all the files that were present in iCubGenova09 because I really don't know if you use it all or part of it.
So I ported also the forearm calibrators called by some XML files you had before.

Anyway, if you do not need them I can surely remove those files 👍🏻 
Give me light green to proceed!

**Comment by S-Dafarra** [comment]
> Ciao @S-Dafarra , I think you refer to forearms calibrators. Honestly, I simply ported all the files that were present in iCubGenova09 because I really don't know if you use it all or part of it. So I ported also the forearm calibrators called by some XML files you had before.
> 
> Anyway, if you do not need them I can surely remove those files 👍🏻 Give me light green to proceed!

Yes please! Yes, those files were already there, but I think this is a nice occasion to avoid having the duplication anymore. I realized the problem only this morning when I actually needed to change some calibration parameters and initially I ended up in the wrong calibration file

**Comment by davidetome** [comment]
@S-Dafarra , done! 👍🏻

**Comment by S-Dafarra** [comment]
> @S-Dafarra , done! 👍🏻

Thanks!

**Comment by pattacini** [comment]
Hi @S-Dafarra 
Do you have other commits to push to this branch?

**Comment by S-Dafarra** [comment]
Actually I rebased this branch on top of ``
devel_iCubGenova09`` altough I haven't tested the single parts xmls. I can open the PR myself later if you don't mind, since those would be the files that were tested

**Comment by pattacini** [comment]
> Actually I rebased this branch on top of ` devel_iCubGenova09` altough I haven't tested the single parts xmls. I can open the PR myself later if you don't mind, since those would be the files that were tested

Fantastic!
So, I'll merge this PR straight away, waiting possibly for your further contributions.
